### ISSUE TITLE
maybe "en" is better than "n" for pinyin input method

### ISSUE THREAD

**Issue by zerovip** [felixonmars/fcitx5-pinyin-zhwiki#13]
Although "n" is the correct pinyin for "嗯", I think people are used to typing "en" in pinyin input method. There is another similar situation for "呣", see [this issue](https://github.com/mozillazg/python-pinyin/issues/109) for reference.

I checked the `zhwiki-20200801.dict.yaml` file, it only involves two words, "呒染挖啊嗯啊" and "嗯哈哈乐团", so it is not a big deal.

Maybe this following codes will help:
```python
# convert.py
# to replace line 64
pinyin_with_n_m = lazy_pinyin(title)
fix_dic = {'n':'en','m':'mu'}
pinyin_fixed = [ fix_dic[i] if i in fix_dic else i for i in pinyin_with_n_m ]
pinyin = _PINYIN_SEPARATOR.join(pinyin_fixed)
```

Thank you for the excellent dictionary, it helps me a lot.

**Comment by ipcjs** [comment]
注音的问题，[用rime的繁体词典](https://github.com/ipcjs/fcitx5-pinyin-zhwiki/releases/tag/20200901)的话，注出来就正常多了：
```
嗯哈哈樂團	en ha ha yue tuan
嗯哈哈樂團	eng ha ha yue tuan
```

**Comment by felixonmars** [comment]
Fixed in 0.2.2.
